.github/workflows/test.yaml

```diff
@@ -27,25 +27,30 @@ jobs:
       # Install Catlin linting tool
       - name: Install linting tools
         run: |
-          set -Eeu
-          GOBIN=/usr/local/bin/ go install github.com/tektoncd/catlin/cmd/catlin@latest
-          GOBIN=/usr/local/bin/ go install github.com/openshift-pipelines/tektoncd-catalog/cmd/catalog-cd@latest
```

Comment:
We still need to run/install `catalog-cd` though (as it is used later)

Reply:
Resolved this issue @vdemeester